bitnami/spring-cloud-dataflow/README.md

```diff
@@ -435,9 +435,10 @@ helm uninstall my-release
 ### RabbitMQ chart parameters
 
 | Name                                      | Description                                                                     | Value       |
-| ----------------------------------------- | ------------------------------------------------------------------------------- | ----------- |
+|-------------------------------------------|---------------------------------------------------------------------------------|-------------|
 | `rabbitmq.enabled`                        | Enable/disable RabbitMQ chart installation                                      | `true`      |
 | `rabbitmq.auth.username`                  | RabbitMQ username                                                               | `user`      |
+| `rabbitmq.service.port`                   | RabbitMQ port if not default                                                    | `5672`      |
```

Comment:
IMHO, I would omit this param here as it belongs to the RabbitMQ subchart, and it is not usually a common setting to change. The `bitnami/spring-cloud-dataflow` chart is configured to pick up this value automatically (although, it's not currently working due to the mentioned regression).

We usually only expose most common/likely to customize params for subcharts, though users can always add more based on the subchart's `values.yaml` file 

https://github.com/bitnami/charts/blob/master/bitnami/rabbitmq/values.yaml